### ISSUE TITLE
Flag conflicts

### ISSUE THREAD

**Issue by aykutfarsak** [snowflakedb/gosnowflake#106]
### Issue description
When i use `flag.Parse()` in my project, it conflicts with glog package.

### Example code
```go
package main

import (
	"flag"
	"fmt"
	"log"
	"os"

	"database/sql"
	_ "github.com/snowflakedb/gosnowflake"
)

const Version = "0.0.1"

func main() {

	var version = flag.Bool("v", false, "prints current program version")

	if *version {
		fmt.Println(Version)
		os.Exit(0)
	}

	db, err := sql.Open("snowflake", "test_user:test_password@test_account/test_db")
	if err != nil {
		log.Fatal(err)
	}

	err = db.Ping()
	if err != nil {
		log.Fatal(err)
	}

	log.Print("Snowflake connection is successfully established.")
}
```

### Error log
```
./main flag redefined: v
panic: ./main flag redefined: v
```

### Configuration
*Driver version (or git SHA)*: 584b6e14f67820b0ccd9a40d3b26bdc31b3b9345

*Go version:* go1.8.1 darwin/amd64

*Client OS:* Darwin Kernel Version 16.6.0

**Comment by smtakeda** [comment]
well, in my understanding, `flag` parameter name space is global, as long as the driver uses `glog`, we cannot avoid the conflict. At the moment, we don't have a plan to change the logging framework, so please use other name for your application.

For long term, the driver may use different logging framework, or if `flag` supports multiple name spaces, we'll leverage that.

Anyway, any suggestion to address this issue is welcome.

**Comment by smtakeda** [comment]
This was addressed by #111. Closing.
